### PR TITLE
Add GitHub ci-workflow action

### PR DESCRIPTION
**Pull request by cmoesel**
The ci-workflow action will:
- install dependencies
- build the app
- run the tests
- run the linter
- check the style (prettier)

This also adds an npm script called `check` that will run the same tests as the ci-workflow.  This allows developers to test their branch locally before pushing to GitHub.

See the run (on this very branch!) here: https://github.com/mcode/covid-19-pui/runs/517517453?check_suite_focus=true

### CODE REVIEW

**Comment by noranda** [comment]
This looks great! Should we add a badge in our readme?

**Comment by cmoesel** [comment]
I can do that.  Also, I assumed that just running it on Linux using Node 12.x was sufficient, but if there is a reason to test it on other OSes or Node versions, let me know.

File: package.json [hunk 1]
```diff
@@ -15,9 +15,13 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
+    "test-no-watch": "react-scripts test --watchAll=false",
     "eject": "react-scripts eject",
     "lint": "eslint './src/**/*.js'",
-    "lint-fix": "eslint './src/**/*.js' --fix"
+    "lint-fix": "eslint './src/**/*.js' --fix",
```
Comment (radamson) [inline package.json]: This is actually my fault, but we should update the `lint` and `lint-fix` command to check `.jsx` files too.

Would you mind making this change in this PR?

Comment (cmoesel) [reply package.json]: Will do.

**Comment by cmoesel** [comment]
@noranda -- see the badge here: https://github.com/mcode/covid-19-pui/blob/github-actions/README.md

File: package.json (hunk 1 above)
Comment (cmoesel) [reply package.json]: Did do.

**Comment by noranda** [comment]
Whoops, it would help to change branches haha. Looks good to me!

**Pull request merged**